submodules/wizard/wizard.js

```diff
@@ -91,7 +91,8 @@ define(function(require) {
 							]
 						}
 					]
-				}
+				},
+				toastTimeout: 10000
```

Comment:
This timeout seems pretty long, double the default time. What type of message are you displaying?

Reply:
I copied the timeout from the old wizard too :sweat_smile: The displayed messages are to notify that the account itself, or an account component was not created/set (i.e. users, add credit, set service plan, limits)

Reply:
Yes please, the default time seems fine.